containers/ecr-viewer/package.json

```diff
@@ -20,7 +20,7 @@
     "format:check": "prettier --check '**/*.{js,json,md,ts,tsx,scss,yaml,yml}'",
     "test": "TZ=America/New_York jest",
     "test:watch": "TZ=America/New_York jest --watch",
-    "test-pg-core": "TZ=America/New_York jest",
+    "test:pg-core": "TZ=America/New_York METADATA_DATABASE_TYPE=postgres METADATA_DATABASE_SCHEMA=core jest --watch --runInBand",
```

Comment:
does this need to do any docker command running? and/or does it rely on something running?